cloud-sql-proxy.yaml

```
@@ -0,0 +1,50 @@
+package:
+  name: cloud-sql-proxy
+  version: 2.7.0
+  epoch: 0
+  description: The Cloud SQL Auth Proxy is a utility for ensuring secure connections to your Cloud SQL instances
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - git
+      - go
+  environment:
+    CGO_ENABLED: 0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/GoogleCloudPlatform/cloud-sql-proxy
+      tag: v${{package.version}}
+      expected-commit: 2527e5aedbca617579a528070e88a6e02c4d88d1
+
+  - uses: go/build
+    with:
+      packages: .
+      output: cloud-sql-proxy
+      ldflags: -w -X github.com/GoogleCloudPlatform/cloud-sql-proxy/v2/cmd.metadataString=container
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Compatibility package to place binaries in the location expected by upstream helm charts
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}
+          ln -sf /usr/bin/cloud-sql-proxy ${{targets.subpkgdir}}/cloud-sql-proxy
+    dependencies:
+      runtime:
+        - cloud-sql-proxy-compat
```

As is this is a dep on itself.
```suggestion
        - cloud-sql-proxy
```

---

thanks, fixed